src/status_im/contexts/wallet/events.cljs

```diff
@@ -227,10 +228,21 @@
 
 (defn get-keypairs-success
   [{:keys [db]} [keypairs]]
-  {:db (assoc-in db [:wallet :keypairs] (data-store/parse-keypairs keypairs))})
+  (let [parsed-keypairs (data-store/parse-keypairs keypairs)
+        default-key-uid (:key-uid (first parsed-keypairs))]
+    {:db (-> db
+             (assoc-in [:wallet :keypairs] parsed-keypairs)
+             (assoc-in [:wallet :selected-keypair-uid] default-key-uid))}))
```

Comment:
I think we have to be careful about the data structure here. I'm not sure if we want to add so many level 2 keys here? Probably it's better to have one level down maybe?
e.g 
(assoc-in [:wallet <some-key> :keypairs] parsed-keypairs)
(assoc-in [:wallet <some-key> :selected-keypair-uid] default-key-uid))}))
I think we've generally kept the top level quite bare otherwise so I think we want to avoid bloating that out too much with many keys - at least within wallet db model.
cc @ulisesmac, @vkjr

Reply:
Maybe `[:wallet :account :keypairs]`?

Reply:
@J-Son89  Yeah, I agree with @briansztamfater although probably  @OmarBasem can suggest a better name if there's a better one.

Reply:
Changed it to `[:wallet :ui :create-account :selected-keypair-uid]`